### ISSUE TITLE
Can you provide the fine-tune configuration file?

### ISSUE THREAD

**Issue by ZooLulu** [lanse-sir/sup#5]
Since we can't find the fine-tune configuration file, we modify the configuration file of train_cave slightly for fine-tune, but the self loss keeps rising, can you provide the fine-tune configuration file

**Comment by lanse-sir** [comment]
Hi, I have uploaded the finetune configuration file.
